CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The change log is available [on GitHub][2].
 0.5.0
 =====
 
+* [#115] (https://github.com/kowainik/tomland/issues/115)
+  Added time combinators to `Toml.BiMap` and `Toml.Bi.Combinators`
 * [#22] (https://github.com/kowainik/tomland/issues/22)
   Report proper type checking error during parsing
```

Comment:
```suggestion
  Report proper type checking error during parsing.
```

Reply:
This suggested change function is very nice :)